config/betterquesting/DefaultQuests/Quests/NoviceThaumaturg-AAAAAAAAAAAAAAAAAAAAFg==/LeadGalenaSilver-AAAAAAAAAAAAAAAAAAACYg==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "You can\u0027t find Lead in the Overworld or the Nether. So it\u0027s time to explore the mysterious Twilight Forest a bit and find a Galena vein at Y 5-40.\n\nYou can get Lead, Silver, Cryolite for Aluminium production and Galena (more lead) in the vein.\n\n§3There\u0027s some Silver in small ores in the OW and Nether, but this is a better source."
+      "desc:8": "You can\u0027t find Lead in the Overworld or the Nether. So it\u0027s time to explore the mysterious Twilight Forest a bit and find a Galena vein at Y 5-40.\n\nYou can get Lead, Silver, Cryolite for Aluminium production and Galena, which is more lead, in the vein.\n\nThere is some Silver in small ores in the OW and Nether, but this one is a better source."
```

Comment:
remove the "one" was better without

Reply:
Fixed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/1f697eeaea591d932268ca7a39f9c73d71eebe01